packs/st2/actions/send_anonymous_install_data.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: "send_anonymous_install_data"
```

Comment:
Naming is the hardest thing. 

I did actually wonder if I should make it generic enough, but then decided to throw this up for comment before I went too much further, whether we'd have a single action that calls home all the time, or actions that fire for each type of data we want to send home (install, pack install, etc)

I still don't know if it's good to stuff too much into a single command.